Actions/WorkflowInitialize/WorkflowInitialize.ps1

```diff
@@ -9,18 +9,43 @@ try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-TestRepoHelper.ps1" -Resolve)
 
-    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
+    # $ENV:GITHUB_ACTION_PATH is
+    #                                      -4       -3         -2       -1
+    # linux:            /home/runner/work/owner/AL-Go-Actions/branch/WorkflowInitialize
+    # windows:  C:\Users\runneradmin\work\owner\AL-Go-Actions\branch\WorkflowInitialize
+    #
+    # For Preview (or Direct AL-Go development branch)
+    #                                      -5    -4     -3     -2       -1
+    # linux:            /home/runner/work/owner/AL-Go/branch/Actions/WorkflowInitialize
+    # windows:  C:\Users\runneradmin\work\owner\AL-Go\branch\Actions\WorkflowInitialize
+    # or:       C:\Users\runneradmin\work\owner\AL-Go\branch\Actions/WorkflowInitialize
+    $ap = "$ENV:GITHUB_ACTION_PATH".Split('/\')
```

Comment:
Just out of curiosity, why not use `github.action_ref` and `github.action_repository` (from https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) instead of splitting the action path? Seems a bit cleaner and less likely to break (in case GitHib go wild and change the action path structure).

Then the check below might not be needed (referring to `if ($branch -eq 'Actions' -and $owner -eq 'AL-Go')`).

As a minimum, consider renaming `$branch` to `$ref` as it's indeed a ref.

Reply:
Good point - these didn't exist earlier but I remember that you also suggested using them in a different situation.
The code became a lot simpler with this, thanks.

Reply:
and... - they are actually not available as ENV variables - they are as github.action_ref and github.action_repository in action.yaml.
Fixed this now - looks much simpler-